python/cuspatial/cuspatial/core/spatial/indexing.py

```diff
@@ -51,7 +62,8 @@ def quadtree_on_points(
             level       : cudf.Series(dtype=np.int8)
                 An int8 column of quadtree levels
             is_quad     : cudf.Series(dtype=np.bool_)
```

Comment:
Should we call this `is_parent` or `is_parent_quad` instead, for clarity? After all, leaves are quads too, right?

Alternatively, call it `is_leaf` and invert the values.

Reply:
I thought about renaming it but didn't. It'd be a breaking change and wasn't sure if we wanted to introduce one.